Gopkg.toml

```diff
@@ -84,3 +88,19 @@ version="v1.4.7"
 [[constraint]]
   name = "github.com/google/go-cmp"
   revision = "6f77996f0c42f7b84e5a2b252227263f93432e9b"
+
+[[constraint]]
```

Comment:
is there any reason why this is 1.12.3 since other dependencies we have is on 1.12.6?

Reply:
> Nice contribution ! Next time please help make the dependency change a separate PR, thanks!

I'm afraid if the dependency is in a separate PR, the unittest will fail due to lack of package. So I make it a separate commit.

Reply:
> is there any reason why this is 1.12.3 since other dependencies we have is on 1.12.6?

I remember when I use 1.12.3, the package sigs.k8s.io/controller-runtime of version "=v0.1.9" cannot be ensured successfully. So I refered to other Kubeflow projects and use 1.12.3